From 935b4d0d95085aa635a09dc7e923a7e88729a61b Mon Sep 17 00:00:00 2001
From: Rui Loura <rui@joyent.com>
Date: Wed, 6 Feb 2019 21:31:03 +0000
Subject: [PATCH] TRITON-1197 sdc-scripts README.md markdown formatting fix
 Reviewed by: Rui <rui@joyent.com> Approved by: Rui <rui@joyent.com>

---
 README.md | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/README.md b/README.md
index fd979e4..0fda376 100644
--- a/README.md
+++ b/README.md
@@ -17,10 +17,10 @@ guidelines](https://github.com/joyent/triton/blob/master/CONTRIBUTING.md) --
 
 The files/scripts here are used for configuring sdc zones.  Currently used are:
 
-setup.sh        - called once when first setting up the zone.
-configure.sh    - called on every zone boot.
-lib/util.sh     - contains common functions for setting up SDC zones.
-etc/root.bashrc - will be loaded as /root/.bashrc in SDC zones.
+* setup.sh: called once when first setting up the zone.
+* configure.sh: called on every zone boot.
+* lib/util.sh: contains common functions for setting up SDC zones.
+* etc/root.bashrc: will be loaded as /root/.bashrc in SDC zones.
 
 The files in the sdc-scripts.git repo are copied in first and then the files
 in the zone's `<repo>.git/boot` directory are copied over.  This way the
-- 
2.21.0

